### PR TITLE
added the hospital and amenities tables and seeds

### PR DESCRIPTION
**Pull request by domikitchen**
## Description ##

Clearly documents the reason for the pull request and what has changed. Delete this test and add your own.
- [ ] This is a bugfix.
- [ ] This is a new feature.
- [ ] This is part of a feature.

## Organization ##

- [ ] Have you linked this pull request to a Trello card?

## Review ##

- [ ] Have you added two reviewers to this pull request?
- [ ] Is this a work in process?
- [ ] Is this ready to be reviewed?

### CODE REVIEW

File: data/migrations/20201109125611_create-hospital-and-amenities-tables.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+exports.up = function(knex) {
+    return knex.schema
+    .raw('CREATE EXTENSION IF NOT EXISTS "uuid-ossp"')
+    .createTable('hospitals', function(table) {
+        table.integer('hospital_id');
+        table.string('hospital_name');
+        table.integer('amenities_id');
```
Comment (Vippsi) [inline data/migrations/20201109125611_create-hospital-and-amenities-tables.js]: I don't think we need the amenities_id as part of the hospital table. I would think we would need the closest_bridge_id instead (Something like that)

File: data/migrations/20201109125611_create-hospital-and-amenities-tables.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        table.float('lat');
+        table.float('long');
+      })
+    
+      .createTable('amenities', function (table) {
+        table.integer('amenities_id');
+        table.string('amenitie_name');
+        table.integer('hospital_id');
```
Comment (Vippsi) [inline data/migrations/20201109125611_create-hospital-and-amenities-tables.js]: I don't think we need the hospital_id as part of the amenities table. I would think we would need the closest_bridge_id instead (Something like that)

File: data/seeds/003_hospitals.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex('hospitals').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('hospitals').insert([
+        {hospital_id: 1, hospital_name: 'Kirehe', amenities_id: "1", lat: -2.267769, long: 30.653169},
```
Comment (Vippsi) [inline data/seeds/003_hospitals.js]: I would change the ids to be 1, 2, 3 so we don't get any errors

**Review by Vippsi — CHANGES_REQUESTED** [review]
Everything looks good, just a couple changes

File: data/seeds/003_hospitals.js (hunk 3 above)
Comment (domikitchen) [reply data/seeds/003_hospitals.js]: thank you for pointing that out

**Review by domikitchen — COMMENTED** [review]
resolved the suggestions

**Review by domikitchen — COMMENTED** [review]
changed the seed ids

**Review by dortega5185 — APPROVED** [review]
Seems all correct to me, great job Dominique.

**Review by Vippsi — COMMENTED** [review]
Everything looks awesome!

**Review by Vippsi — APPROVED** [review]
Awesome!

**Pull request merged**